libs/designsystem/src/lib/components/angular-component-lib/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { fromEvent } from 'rxjs';
+
```

Comment:
Not related to specific line: 
Seems like the path for this (angular-component-lib) is not configurable. It is autogenerated by stencil, based on where the proxies.ts file is outputted.